README.md

```diff
@@ -21,6 +21,26 @@ For more info, see:
 1. `npm install`
 2. `./node_modules/.bin/web-ext run -s src/`
 
+or
+
```

Comment:
@groovecoder and @maxxcrawford  just thought on providing some additional ways to develop. I found the second approach quite easy and maybe it would help someone. however I thought it would open to a separate PR.

Reply:
I intended to send the modifications of the README as a separate PR but for some reason it automatically gets merged with this PR. Is it not possible to send a  new  PR with different changes until this PR is closed?

Reply:
removed the changes in README for now.

Reply:
removed the changes in README for now.